lint/witnesslint/linter.py

```diff
@@ -991,9 +988,9 @@ def _exit(exit_code=None):
     sys.exit(exit_code)
 
 
-def main(argv):
+def main():
```

Comment:
As far as I am aware it is a best practice to not force the use of `sys.argv` within the `main` method, because this makes it impractical to use the method from somewhere else (tests, other scripts).

If it is required that this is callable without arguments, the standard pattern is
```python
def main(argv=None):
    if argv is None:
        argv = sys.argv
```

Reply:
Right, makes sense. Adjusted it to use the pattern you mentioned, but with `argv = sys.argv[1:]` instead. This is because other scripts should not pass the script name, so we don't want to throw away arguments later.